prow/config/jobs_test.go

```diff
@@ -118,9 +114,6 @@ func TestPresubmits(t *testing.T) {
 					if job.Context == job2.Context {
 						t.Errorf("Jobs %s and %s have the same context: %s", job.Name, job2.Name, job.Context)
 					}
```

Comment:
Colliding contexts is not an error. That is one of the main use cases for the `context` field!

Reply:
this prohibits colliding contexts for jobs with different names, I didn't add this.

Reply:
Yes, we can fix this in another PR, but it is still a bug. One of the intended uses of `context` is to allow jobs with different names to report under the same context. That is how we unique job names would be achieved.